core/src/main/java/org/apache/gravitino/credential/CatalogCredentialContext.java

```diff
@@ -35,4 +35,27 @@ public CatalogCredentialContext(String userName) {
   public String getUserName() {
     return userName;
   }
+
+  @Override
+  public int hashCodeIgnoreUser() {
+    return 9999;
+  }
+
+  @Override
+  public boolean equalsIgnoreUser(Object o) {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || !(o instanceof CatalogCredentialContext)) {
+      return false;
+    }
+    return true;
```

Comment:
Do we need more checks before returning `true`?

Reply:
removed